.github/workflows/flaky-test.yml

```diff
@@ -0,0 +1,28 @@
+name: Flaky Test Finder
```

Comment:
If you wanted to simplify this into a verb, I've seen this called `Deflake` or `--deflake` before.